Puppetfile.lock

```diff
@@ -129,21 +113,12 @@ GIT
       puppetlabs-stdlib (>= 2.4.0)
 
 DEPENDENCIES
+  camptocamp-systemd (< 2.0.0, >= 1.0.0)
   katello-candlepin (< 5.2.0, >= 5.1.1)
   katello-certs (< 4.3.0, >= 4.2.0)
   katello-foreman_proxy_content (< 7.2.0, >= 7.1.1)
   katello-katello (< 8.1.0, >= 8.0.1)
   katello-pulp (< 5.5.0, >= 5.4.2)
   katello-qpid (< 4.3.0, >= 4.2.0)
-  puppetlabs-mysql (>= 4.0.0)
-  puppetlabs-postgresql (>= 5.6.0)
-  puppetlabs-puppetdb (>= 0)
-  theforeman-dhcp (< 4.2.0, >= 4.1.1)
-  theforeman-dns (< 5.3.0, >= 5.2.0)
-  theforeman-foreman (< 9.3.0, >= 9.2.0)
-  theforeman-foreman_proxy (< 7.3.0, >= 7.2.1)
-  theforeman-git (< 4.1.0, >= 4.0.0)
-  theforeman-puppet (< 9.1.0, >= 9.0.0)
-  theforeman-tftp (< 4.2.0, >= 4.1.0)
```

Comment:
I think you want to put these back

Reply:
jaja, I thought librarian would be smarter than this ;)